java/ql/lib/ext/java.io.model.yml

```diff
@@ -47,6 +47,8 @@ extensions:
       - ["java.io", "RandomAccessFile", False, "writeUTF", "", "", "Argument[0]", "write-file", "manual"]
       - ["java.io", "Writer", True, "append", "", "", "Argument[0]", "write-file", "manual"]
       - ["java.io", "Writer", True, "write", "", "", "Argument[0]", "write-file", "manual"]
+      - ["java.io", "OutputStream", True, "write", "(byte[])", "", "Argument[0]", "write-file", "manual"]
+      - ["java.io", "OutputStream", True, "write", "(byte[],int,int)", "", "Argument[0]", "write-file", "manual"]
```

Comment:
These should be handled by the model above:

```yml
      - ["java.io", "FileOutputStream", False, "write", "", "", "Argument[0]", "write-file", "manual"]
```

We try not to model `[Input|Output]Stream` generally as file operations, because they could be of other types (like `ByteArrayOutputStream`). That's why we limit ourselves to `FileOutputStream.write` in this case.

Did you spot a FN caused by this? Sometimes the lack of type information in the analysis can lead to a FN (i.e. the stream is indeed a `FileOutputStream` but the analysis sees it only as `OutputStream`).

Reply:
This makes sense. I will remove and update the models right now